### ISSUE TITLE
The "Start new calculation" button gets blocked by a pop-up blocker

### ISSUE THREAD

**Issue by edan-bainglass** [aiidalab/aiidalab-qe#988]
At least on Mac OS with Safari.
Maybe just replacing the button with a `<a target="_blank">` would work? These should not be blocked?

Moreover (as discussed): put it as a fourth gray (not green) button after "Job History". Then call it "New calculation (in new tab)".

Duplicate of #891
